pyproject.toml

```diff
@@ -94,7 +94,7 @@ jsonschema = "~3.2"
 packaging = "~19.0"
 gevent = ">=1.4,<1.6"
 pytz = "^2019.1"
-rasa-sdk = "^1.9.0"
+rasa-sdk = "^1.10.0"
```

Comment:
@alwx That's not needed unless there are incompatibilities, right? Might remove this part in the future.